modules/ocs2/src/main/scala/gem/ocs2/DoobieClient.scala

```diff
@@ -16,18 +16,16 @@ import scalaz.effect.IO
 /** Shared support for import applications using Doobie. */
 trait DoobieClient {
 
+  val Url  = "jdbc:postgresql:gem"
+  val User = "postgres"
+  val Pass = ""
```

Comment:
We should change everything to use a `gem` user but I don't think it's important right now.